public/src/map.js

```diff
@@ -22,10 +22,27 @@ export async function initMap(apiKey) {
     maxBounds: ukBounds
   });
 
-  const geocoder = new MapboxGeocoder({
+  // const areaGeocoder = new MapboxGeocoder({
+  //   accessToken: apiKey,
+  //   mapboxgl: mapboxgl,
+  //   zoom: 18,
+  //   placeholder: 'Search UK Places',
+  //   bbox: ukSearchBounds,
+  //   proximity: {
+  //     longitude: long,
+  //     latitude: lat
+  //   },
+  //   marker: false
+  // });
+  
+  const constituencyGeocoder = new MapboxGeocoder({
     accessToken: apiKey,
+    localGeocoder: dummy,
```

Comment:
What is this localGeocoder?

Reply:
On my side, at least, `localGeocoder` seemed to be whatever thats built into the package and available locally, if `localGeocoder = localSearch`, then it will return this error 
![image](https://user-images.githubusercontent.com/65177718/112053774-a9f34580-8b4c-11eb-968b-9fa2d5514363.png)

Hence stumbled upon some solution online